src/site/stages/build/process-cms-exports/schemas/output/media-image.js

```diff
@@ -8,7 +8,23 @@ module.exports = {
     entityMetatags: { $ref: 'MetaTags' },
     fieldMediaInLibrary: { type: 'boolean' },
     fieldOwner: { $ref: 'output/taxonomy_term-administration' },
-    image: { $ref: 'output/file' },
+    // image: { $ref: 'output/file' },
```

Comment:
Why is this no longer `output/file`? I see it has `derivative`, but `file` doesn't now. Can you help me understand this change?

Reply:
Thanks for catching this. I actually need to update the `image` field in this schema. I removed `derivative` from the `file` transformer because the only entity that has `derivative` is `media-image`. So I figured we should move it out of `file` because other transformers like `media-document` use `file` but don't have the `derivative` field.